Makefile

```diff
@@ -95,6 +97,19 @@ check-go-format: prebuild-check
 	&& exit 1 \
 	|| true
 
+.PHONY: golint
+.ONESHELL: golint
+ ## Run different static code analysis for go
+golint: clean-golint $(GOIMPORTS_BIN) $(GOLINT_BIN) $(GOCYCLO_BIN)
+	@echo "--- GoCYCLO CODE ANALYSIS ----" >> $(GOLINT_TMP_FILE);
+	@$(foreach d,$(GOLINT_DIRS),$(GOCYCLO_BIN) -over 15 $d | grep -vEf .golint_exclude >> $(GOLINT_TMP_FILE);)
```

Comment:
@kwk We're going to print it into the stdout so there isn't need to redirect it to a file